bundler.go

```diff
@@ -64,8 +64,11 @@ type Configuration struct {
 
 // ConfigurationEnvironment represents the bundle configuration environment
 type ConfigurationEnvironment struct {
-	Arch string `json:"arch"`
-	OS   string `json:"os"`
+	Arch        string `json:"arch"`
+	OS          string `json:"os"`
+	CC          string `json:"CC"`
```

Comment:
Instead of listing each and every possible env variable at the root of this struct, I'd rather have a unique `EnvironmentVariables map[string]string` at the root of this struct (and attributes should be ordered alphabetically).